client/src/components/GameArea.js

```diff
@@ -43,30 +61,286 @@ class GameArea extends Component {
+                  "Card text here"
+                )}
+                {this.addCardToTopBattlefield(
+                  "Sonic Assault",
+                  "{1}{U}{R}",
+                  "https://img.scryfall.com/cards/art_crop/front/c/c/cc61a398-cf16-415b-b3cf-897217dc7cc9.jpg?1538880557",
+                  "Instant",
+                  "https://img.scryfall.com/sets/grn.svg?1545022800",
+                  "Card text here"
+                )}
+                {this.addCardToTopBattlefield(
+                  "Sonic Assault",
+                  "{1}{U}{R}",
+                  "https://img.scryfall.com/cards/art_crop/front/c/c/cc61a398-cf16-415b-b3cf-897217dc7cc9.jpg?1538880557",
+                  "Instant",
+                  "https://img.scryfall.com/sets/grn.svg?1545022800",
+                  "Card text here"
+                )}
+
+              </Col>
+            </Row>
+
+            {/* Bottom row of cards */}
+            <Row
+              className="battlefield-bottom-row p-0 m-0 mh-50 h-50"
+              style={
+                {
+                  "overflow": "auto"
+                }
+              }
+            >
+              <Col
+                
+                className="battlefield-bottom d-inline-flex flex-wrap border justify-content-start card-row card-row-top my-0 mx-0 px-0"
+              >
+
+                {this.addCardToTopBattlefield(
+                  "Sonic Assault",
+                  "{1}{U}{R}",
+                  "https://img.scryfall.com/cards/art_crop/front/c/c/cc61a398-cf16-415b-b3cf-897217dc7cc9.jpg?1538880557",
+                  "Instant",
+                  "https://img.scryfall.com/sets/grn.svg?1545022800",
+                  "Card text here"
+                )}
+
+                {this.addCardToTopBattlefield(
+                  "Sonic Assault",
+                  "{1}{U}{R}",
+                  "https://img.scryfall.com/cards/art_crop/front/c/c/cc61a398-cf16-415b-b3cf-897217dc7cc9.jpg?1538880557",
+                  "Instant",
+                  "https://img.scryfall.com/sets/grn.svg?1545022800",
+                  "Card text here"
+                )}
+
+              </Col>
+            </Row>
+
+
+          </Col>
+
+
+
+          {/* Sidebar area.  */}
+
+
+          {/* Sidebar for exile,graveyard,hand,library  */}
+          <Col
+            xs="2"
+            className="side-bar-col flex-shrink-3 px-0"
+            style={
+              {
+                "max-height": '100%'
+              }
+            }
+          >
+            <Sidebar></Sidebar>
+          </Col>
+
+        </Row>
+
+      </Container >
+    );
+  }
+
+  mainTitle() {
+    return (
+      <Jumbotron className="d-none d-sm-flex mh-100 pt-0 pb-0 mb-0">
+        <div className="mh-100">
+          <h6 className="display-5 mh-100">Main Board</h6>
+          <p className="d-none d-sm-flex">View of your battlefield.</p>
         </div>
-      </div>
+      </Jumbotron>
+    );
+  }
+
+  lifeComponent() {
```

Comment:
We should eventually break this out into its own component.

Reply:
I was going to put buttons in the top nav bar to have a simple life counter for yourself, but we'll have to come up with an idea for how we want to deal with life counts, your own and others' in the game.

Reply:
I think on our personal device, we just show our own life total, and on the big screen, we can include everyone's life (and counters and/or commander damage).